### ISSUE TITLE
support for Atom (or any other text editor)

### ISSUE THREAD

**Issue by jleung51** [CtrlShiftGo/utilities#12]
Currently, the *Journal* program opens the file in Sublime Text only (program calls are [here](https://github.com/CtrlShiftGo/utilities/blob/56197e2fb598001dc6714b74b206ffeb177f4243/journal.cpp#L122) and [here](https://github.com/CtrlShiftGo/utilities/blob/56197e2fb598001dc6714b74b206ffeb177f4243/journal.cpp#L179)).

This should be changed to enable support for configuring the journal to open in Atom, Vim, or any other text editor (except Emacs, of course).

**Comment by CtrlShiftGo** [comment]
Good idea! I have a branch to implement support for atom. The only issue may be that different editors will open to specific line numbers differently/not at all. I haven't looked into editors aside from atom and sublime.
